### ISSUE TITLE
Using Depreciated feature of Selenium

### ISSUE THREAD

**Issue by iamAbhishekkumar** [chaudharypraveen98/Amazon-Product-Scraper#3]
![image](https://user-images.githubusercontent.com/53290881/215135725-d4de1d72-8118-4273-80d4-ece292e5771a.png)

from 4.3, find_element_by_css_selectors and all other functions has been remoed.

Solution : 
Change the fucntions accordingly to new conventions.

**Comment by iamAbhishekkumar** [comment]
If you want I will fix by tommorrow. :smile:

**Comment by chaudharypraveen98** [comment]
Yeah sure take your time. Actually it was a long ago project in my local system. So thought of pushing to GitHub.

**Comment by iamAbhishekkumar** [comment]
Oh....i see...😅
